src/main/java/uk/gov/hmcts/reform/prl/handlers/TransferToAnotherCourtEventHandler.java

```diff
@@ -0,0 +1,165 @@
+        CaseData caseData = event.getCaseData();
+        if (caseData.getCourtEmailAddress() != null) {
+            sendTransferCourtEmail(caseData);
+            sendTransferToAnotherCourtEmail(event.getAuthorisation(),caseData);
+        }
+    }
+
+    private void sendTransferToAnotherCourtEmail(String authorization,CaseData caseData) {
+        try {
+            sendgridService.sendTransferCourtEmailWithAttachments(authorization,
+                                                                  getEmailProps(caseData),
+                                                                  caseData.getCourtEmailAddress(), getAllCaseDocuments(authorization,caseData));
+        } catch (IOException e) {
+            log.error("Failed to send Email");
+        }
+    }
+
+    private void sendTransferCourtEmail(CaseData caseData) {
+        emailService.send(
+            caseData.getCourtEmailAddress(),
+            EmailTemplateNames.TRANSFER_COURT_EMAIL_NOTIFICATION,
+            transferCaseContentProvider.buildCourtTransferEmail(caseData,getConfidentialityText(caseData)),
+            LanguagePreference.english
+        );
+    }
+
+    private Map<String, String> getEmailProps(CaseData caseData) {
+        Map<String, String> combinedMap = new HashMap<>();
+        combinedMap.put("caseNumber", String.valueOf(caseData.getId()));
+        combinedMap.put("caseName", caseData.getApplicantCaseName());
+        combinedMap.put("issueDate", CommonUtils.formatDate(D_MMMM_YYYY, caseData.getIssueDate()));
+        combinedMap.put("applicationType", caseData.getCaseTypeOfApplication());
+        combinedMap.put("confidentialityText", getConfidentialityText(caseData));
+        combinedMap.put("courtName", caseData.getTransferredCourtFrom());
+        combinedMap.putAll(getCommonEmailProps());
+        return combinedMap;
+    }
+
+    private String getConfidentialityText(CaseData caseData) {
+        return isCaseContainConfidentialDetails(caseData)
+            ? "This case contains confidential contact details." : "";
+    }
+
+    private boolean isCaseContainConfidentialDetails(CaseData caseData) {
+        return CollectionUtils.isNotEmpty(caseData.getApplicantsConfidentialDetails())
+            || CollectionUtils.isNotEmpty(caseData.getRespondentConfidentialDetails())
+            || CollectionUtils.isNotEmpty(caseData.getChildrenConfidentialDetails());
+    }
+
+    private Map<String, String> getCommonEmailProps() {
+        Map<String, String> emailProps = new HashMap<>();
+        emailProps.put("subject", "A case has been transferred to your court");
+        emailProps.put("content", "Case details");
+        emailProps.put("attachmentType", "pdf");
+        emailProps.put("disposition", "attachment");
+        return emailProps;
+    }
+
+    private List<Document> getAllCaseDocuments(String authorisation, CaseData caseData) {
+
+        List<Document> documentList = new ArrayList<>();
```

Comment:
we had this implemented in Send and Reply. we could have reused that logic instead of adding same logic again.

Reply:
@yogendraU  as discussed logic in send and reply doesn't contain document object so we can't use it in our scenario.